substrafl/remote/register/generate_wheel.py

```diff
@@ -138,7 +138,7 @@ def pypi_lib_wheels(lib_modules: List, operation_dir: Path, python_major_minor:
                 )
             except subprocess.CalledProcessError as e:
                 raise ConnectionError(
-                    "Couldn't access to Owkin pypi, please ensure you have access to https://pypi.owkin.com/simple/.",
+                    "Couldn't access to pypi, please ensure you have access to https://pypi.org",
```

Comment:
maybe we can remove the try except now, no need to try catch to say to the user they don't have access to pypi

Reply:
yup